Assets/BossRoom/Scripts/Client/UI/QualityButton.cs

```diff
@@ -5,7 +5,8 @@ namespace Unity.Multiplayer.Samples.BossRoom.Client
 {
     public class QualityButton : MonoBehaviour
     {
-        public TMP_Text qualityBtnText;
+        [SerializeField]
+        private TMP_Text qualityBtnText;
```

Comment:
```suggestion
        TMP_Text m_QualityButtonText;
```
Will have to update references here.